internal/handlemap/handle_map.go

```diff
@@ -55,6 +55,10 @@ const (
 	HandleFlagDirty          // File has been modified with write operation or is a new file
 	HandleFlagFSynced        // User has called fsync on the file explicitly
 	HandleFlagCached         // File is cached in the local system by blobfuse2
+	// Following are the Dcache Flags
+	HandleFSAzure
+	HandleFSDcache
+	HandleFlagDcacheAllowWrites // Handle can only write If this flag is set
```

Comment:
HandleFSAzure               // Handle refers to a file in Azure
	HandleFSDcache              // Handle refers to a file in Distributed Cache.
	                            // Both HandleFSAzure and HandleFSDcache will be set for handles corresponding
	                            // to file paths without an explicit fs=azure/fs=dcache namespace specified.
	HandleFlagDcacheAllowWrites // Write to Distributed Cache through this handle is only allowed if this flag is set
	HandleFlagDcacheAllowReads  // Read from Distributed Cache through this handle is only allowed if this flag is set